### ISSUE TITLE
Enable user to facet by component terms of a concept (from 814)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-marklogic#27]
**Problem Description**
Users cannot filter results by concepts on concept tabs. 

Ex:
In the following search: https://lux-front-tst.collections.yale.edu/view/results/concepts?q=%7B%22AND%22%3A%5B%7B%22name%22%3A%22banjo%22%7D%5D%7D
one of the results has the following title: Banjo--Methods--Self-instruction

**Expected Behavior/Solution**: 
Users should be able to filter results by concepts with the component term `methods` and/or with the component term `self-instruction` on Concepts tab. 

Marklogic will provide a facet for Concept Influenced By

**Requirements:** 
Once ML is complete, New facet 'Component Terms' will appear under Concept tab and list check box options of Component Terms. (See FE ticket) 

**Needed for Promotion:** 
- [x] Front-end ticket to add facet https://github.com/project-lux/lux-frontend/issues/68

**UAT/Lux Examples:** 
https://lux-front-tst.collections.yale.edu/view/results/concepts?q=%7B%22AND%22%3A%5B%7B%22name%22%3A%22banjo%22%7D%5D%7D

**Dependencies/Blocks:**
- Needed for - https://github.com/project-lux/lux-frontend/issues/68

**Related Github Issues**

**Related Links:** 

~**Wireframe/Mockup:**** ~

**Comment by clarkepeterf** [comment]
@jffcamp @prowns 

There are multiple types of records that can influence a concept as a component term Agents, Concepts, Events, and Places - we already have search terms for each of these types separately:
<img width="804" alt="Screenshot 2024-06-20 at 2 18 35 PM" src="https://github.com/project-lux/lux-marklogic/assets/59657152/7fb9380c-7c48-40e6-bdf7-9c255934e2f0">

We could use the same fields to make separate facets - or we can create a facet that combines all types - it just requires a new index.

I'm guessing we want the combined facet? Or are individual facets preferred?

**Comment by jffcamp** [comment]
Please move forward with the individual facets. If we need a combined facet, that can be a new ticket.

**Comment by clarkepeterf** [comment]
I've discovered we already have facet configuration for these individual facets:

conceptInfluencedByAgentId
conceptInfluencedByConceptId
conceptInfluencedByEventId
conceptInfluencedByPlaceId

Ready for consumption by the frontend
